jupyterhub-k8s-network-tools.yaml

```diff
@@ -0,0 +1,35 @@
+Creates a package which represents the following upstream image:
```

Comment:
Something got messed up here - I think some feedback ended up being a code suggestion. Once this is fixed it should pass CI

Reply:
This should work 🤞🏽